.github/workflows/build_main_documentation.yml

```diff
@@ -119,12 +119,12 @@ jobs:
           pip install black
           cd ..
 
-      - name: Make Graphcore documentation
-        run: |
-          cd optimum-graphcore
-          make doc BUILD_DIR=graphcore-doc-build VERSION=${{ env.VERSION }}
-          sudo mv graphcore-doc-build ../optimum
-          cd ..
+      #- name: Make Graphcore documentation
+        #run: |
+          #cd optimum-graphcore
+          #make doc BUILD_DIR=graphcore-doc-build VERSION=${{ env.VERSION }}
+          #sudo mv graphcore-doc-build ../optimum
+          #cd ..
```

Comment:
Same.
I'm pretty sure you only commented for testing purposes but just wanted to double check.